setup/src/opts.ts

```diff
@@ -70,9 +73,11 @@ export function getOpts(
   inputs: Record<string, string>
 ): Options {
   core.debug(`Inputs are: ${JSON.stringify(inputs)}`);
-  const stackNoGlobal = (inputs['stack-no-global'] || '') !== '';
-  const stackSetupGhc = (inputs['stack-setup-ghc'] || '') !== '';
-  const stackEnable = (inputs['enable-stack'] || '') !== '';
+  const isEnabled = (s: string): boolean => (inputs[s] || '') !== '';
+  const readList = (s: string): string[] => (inputs[s] || '').split('\n');
```

Comment:
Should we filter empty lines out here? I think (but haven't verified) that

```yaml
key: |
 a
 b
```

would result in `['', 'a', 'b']` otherwise.